Quickstart.md

```diff
@@ -0,0 +1,197 @@
+from appcli.models.configuration import Configuration
+from appcli.orchestrators import DockerComposeOrchestrator
+
+# ------------------------------------------------------------------------------
+# CONSTANTS
+# ------------------------------------------------------------------------------
+
+# directory containing this script
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+
+# ------------------------------------------------------------------------------
+# PRIVATE METHODS
+# ------------------------------------------------------------------------------
+
+def main():
+    configuration = Configuration(
+        app_name='myapp',
+        docker_image='brightsparklabs/myapp',
+        seed_app_configuration_file=Path(BASE_DIR, 'resources/settings.yml'),
+        stack_configuration_file=Path(BASE_DIR, 'resources/stack-settings.yml'),
+        baseline_templates_dir=Path(BASE_DIR, 'resources/templates/baseline'),
+        configurable_templates_dir=Path(BASE_DIR, 'resources/templates/configurable'),
+        orchestrator=DockerComposeOrchestrator(
+            docker_compose_file = Path('docker-compose.yml')
+        ),
+        
+    )
+    cli = create_cli(configuration)
+    cli()
+
+# ------------------------------------------------------------------------------
+# ENTRYPOINT
+# ------------------------------------------------------------------------------
+
+if __name__ == '__main__':
+    main()
+```
+
+Make sure the myapp.py has executable permission. In order to do this, you need to cd into the directory of myapp.py in your terminal and execute the following command
+
+```
+chmod +x myapp.py
+```
+
+### Dockerfile Setup
+
+Outside of the src folder, create a docker file and paste in the following settings so we can assemble our image
+
+```Dockerfile
+# filename: Dockerfile
+
+FROM brightsparklabs/appcli
+
+ENTRYPOINT ["./myapp.py"]
+WORKDIR /app
+
+# install compose if using it as the orchestrator
+RUN pip install docker-compose
+
+COPY requirements.txt .
+RUN pip install --requirement requirements.txt
+COPY src .
+
+ARG APP_VERSION=latest
+ENV APP_VERSION=${APP_VERSION}
+```
+
+We can see in the dockerfile, it is referring to a requirements.txt file which we will need to create. Go ahead and create a requirements.txt file in the same location as your Dockerfile and enter in the following `bsl-appcli==1.3.4` 
+This will install the appcli package in the image.
+
+### Create a docker compose file
+You will need to create a docker compose file which will be placed in the baseline folder. Give it the default name docker-compose.yml
+
+We will be using an Echo-Server for our example application which is useful for testing purposes as it replicates the request sent by the client and sends it back. More information on this container can be found [here](https://ealenn.github.io/Echo-Server/pages/quick-start/docker.html#run)
+
+Paste in the following code into your docker compose file.
+
+```Dockerfile
```

Comment:
This is a docker-compose file, not a Dockerfile.
See the different supported languages for syntax highlighting here:
https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Reply:
Doesn't look like docker-compose is supported, so probably just worth changing to 'yaml'